lib/perl/Genome/Disk/Command/Allocation/Unarchive.pm

```diff
@@ -59,6 +59,11 @@ sub _link_allocation_to_analysis_project {
         unless($owner->model->analysis_project) {
             $self->fatal_message('No analysis project set on model for build %s.  Please use `genome analysis-project add-model` to correct this.', $owner->__display_name__);
         }
+    } elsif ( $owner->isa('Genome::InstrumentData::Imported') ) {
+        $self->analysis_project->add_analysis_project_bridge(
```

Comment:
There's a unique constraint on this bridge table, so we'll need to make sure we don't try to add the same instrument data to the same analysis project twice.  (`Genome::SoftwareResult::User` is an `ObjectWithLockedConstruction`, so it will find the existing object when `add_user` is called a second time, but this bridge entity is not.)